README.md

```diff
@@ -300,7 +300,7 @@ This project is in the worldwide [public domain](LICENSE.md).
 
 This project is in the public domain within the United States, and
 copyright and related rights in the work worldwide are waived through
-the [CC0 1.0 Universal public domain
+the [CC0.1.0-rc.1 Universal public domain
```

Comment:
I'm not sure if you want to rebase and force push to undo this but it looks like this got caught by the `bump_version.sh` script.

```suggestion
the [CC0 1.0 Universal public domain
```

Reply:
It looks like `0.1.0` of the old version was interpreted as `0<any character>1<any character>0` by `sed`. We should probably create an issue for that because it will likely come up again in other projects.

Reply:
Good catch!

Reply:
Please have a look at cisagov/skeleton-docker#122.  Once @mcdonnnj and @dav3r are happy with those changes and approve that PR then I will go ahead and create the others.